lib/lightning_web/live/workflow_live/ai_assistant_component.ex

```diff
@@ -584,6 +584,59 @@ defmodule LightningWeb.WorkflowLive.AiAssistantComponent do
     """
   end
 
+  attr :content, :string, required: true
+  attr :attributes, :map, default: %{}
+
+  def formatted_content(assigns) do
+    assistant_messages_attributes = %{
+      "a" => %{
+        class: "text-primary-400 hover:text-primary-600",
+        target: "_blank"
+      },
+      "h1" => %{class: "text-2xl font-bold mb-6"},
+      "h2" => %{class: "text-xl font-semibold mb-4 mt-8"},
+      "ol" => %{class: "list-decimal pl-8 space-y-1"},
+      "ul" => %{class: "list-disc pl-8 space-y-1"},
+      "li" => %{class: "text-gray-800"},
+      "p" => %{class: "mt-1 mb-2 text-gray-800"}
+    }
+
+    merged_attributes =
+      Map.merge(assistant_messages_attributes, assigns.attributes)
+
+    assigns =
+      case Earmark.Parser.as_ast(assigns.content) do
+        {:ok, ast, _} ->
+          process_ast(ast, merged_attributes) |> raw()
+
+        _ ->
+          assigns.content
+      end
+      |> then(&assign(assigns, :content, &1))
+
+    ~H"""
+    <article><%= @content %></article>
```

Comment:
@elias-ba I made a small change here, I moved the `raw` call to the result of `process_ast`, because in the rare chance the parser doesn't do anything to the output - we shouldn't assume it's safe and should render it out as `{:safe, str}` (which is what `raw` does).